src/compat/posix/time/strptime.c

```diff
@@ -0,0 +1,320 @@
+/**
+ * @file strptime.c
+ * @brief writing linux function strptime for embox os
+ *
+ * author Tamara Deryugina <tamara.deryugina@gmail.com>
+ * @date 02.03.2020
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <ctype.h>
+#include <math.h>
+
+static int num_length(const char *s) {
+	int count = 0;
+	while ((s[count] >= '0') && (s[count] <= '9')) {
+		count++;
+	}
+	return count;
+}
+
+static int word_length(const char *s) {
+	int count = 0;
+	while ((s[count] >= 'A') && (s[count] <= 'z')) {
```

Comment:
This is incorrect check. For example, `[` would be interpreted as a alphabetical symbol here.

Reply:
Done